### PR TITLE
Dockerize Bassa Core

### PR DESCRIPTION
**Pull request by kmehant**
@agentmilindu Updated the dockerfile as per the given instructions in the GCI task. Kindly Review.

### CODE REVIEW

**Comment by kmehant** [comment]
@agentmilindu All tests passed :) Kindly review.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM python:3
+MAINTAINER SCoRe Lab Community <commuity@scorelab.org>
+
+RUN apt-get update
+
+RUN mkdir -p /home/Bassa
+RUN apt-get install git
+RUN git clone https://github.com/scorelab/Bassa.git /home/Bassa
```
Comment (agentmilindu) [inline Dockerfile]: Hi @kmehant, we don't need cloning the repo with git here. We can simply copy the files from the local repo. 

You can refer: https://docs.docker.com/engine/reference/builder/#copy

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+FROM python:3
+MAINTAINER SCoRe Lab Community <commuity@scorelab.org>
+
+RUN apt-get update
+
+RUN mkdir -p /home/Bassa
```
Comment (agentmilindu) [inline Dockerfile]: You can set `/home/Bassa` as the `WORKDIR`, then you can simply say, 

`RUN chmod 755 setup.sh`
`RUN setup.sh`

Refer: https://docs.docker.com/engine/reference/builder/#workdir

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+RUN apt-get install git
+RUN git clone https://github.com/scorelab/Bassa.git /home/Bassa
+RUN chmod 755 /home/Bassa/setup.sh
+RUN /home/Bassa/setup.sh
```
Comment (agentmilindu) [inline Dockerfile]: One good practice writing dockerfiles is combining the RUN commands together if they relate. 

You can do as follows, 

```
RUN apt-get update \
&&  chmod 755 /home/Bassa/setup.sh && \
&&  /home/Bassa/setup.sh && \
&&  python components/core/setup.py develop
```

Refer this : https://hackernoon.com/tips-to-reduce-docker-image-sizes-876095da3b34

File: Dockerfile (hunk 1 above)
Comment (kmehant) [reply Dockerfile]: @agentmilindu I have a doubt here, can i say

```
WORKDIR /home/Bassa
COPY Bassa WORKDIR 
```

Is this valid?

Comment (agentmilindu) [reply Dockerfile]: You can say `COPY . /home/Bassa `

http://blog.cloud66.com/how-to-get-code-into-a-docker-container/#option3usingtheaddorcopycommand

**Comment by kmehant** [comment]
@agentmilindu Thank you for the guidance, i addressed all the changes :)

**Pull request merged**